### DISCUSSION TITLE
Unable to install

### DISCUSSION THREAD

**Discussion by LeonardoG67** [givanz/Vvveb#201] [Q&A]
I can't install the latest version of April 2024. I use xampp and I configured everything as described in the installation section but at the end it gives me an error. I would like to install it in the Onfernovveb subfolder so my path is localhost:808/Onfernovvveb/
![Screenshot 2024-10-19 112927](https://github.com/user-attachments/assets/88a47ba9-97ec-43ef-a335-4f4d7d30d041)

**Comment by givanz** [comment]
Thanks for the bug report, it's fixed in the last commit https://github.com/givanz/Vvveb/commit/964064e4d7d090b6cce5e7bd4a50563dbfe18649 

Changed files [/install/controller/index.php](https://github.com/givanz/Vvveb/blob/master/install/controller/index.php) and [/system/functions.php](https://github.com/givanz/Vvveb/blob/master/system/functions.php)

After updating the files you can delete `/config/db.php` to reset the installation.

**Reply by LeonardoG67** [reply]
installation completed!!
but there are other problems with the paths: are they correct or incorrect a bit everywhere
<img width="343" alt="Snap325" src="https://github.com/user-attachments/assets/e6cedc06-2859-479b-b329-95db8159fb82">
<img width="339" alt="Snap326" src="https://github.com/user-attachments/assets/1cf3872d-51db-40ae-af8f-1c8cbf1b4892">

**Comment by givanz** [comment]
Missed the admin path bug, please update [/system/functions.php](https://github.com/givanz/Vvveb/blob/master/system/functions.php) again it's fixed in the last commit https://github.com/givanz/Vvveb/commit/eb682965ad2022c219e59e36ebeac381fdab567a

**Reply by LeonardoG67** [reply]
the same condition, but only in the new release:

**Warning
: Trying to access array offset on value of type bool in
D:\Leonardo\Tools\xampp\htdocs\Onfernovvveb\admin\controller\editor\editor.php
on line
200**

![Screenshot 2024-10-20 101622](https://github.com/user-attachments/assets/245bf76f-7045-4f06-bf7a-eb8161ac6ecc)

**Reply by givanz** [reply]
Thanks for the bug report, it's fixed in the last commit https://github.com/givanz/Vvveb/commit/a3ba3de70997ea47a71d401489519edeca1f9731

Changed file [/admin/controller/editor/editor.php](https://github.com/givanz/Vvveb/blob/master/admin/controller/editor/editor.php)
